### DISCUSSION TITLE
Add category for list of community plugins

### DISCUSSION THREAD

**Discussion by stevedylandev** [wevm/frog-archived#29] [Q&A] [answered]
Hey there! We are just about finished with our first plugin that can be used with frog and we were curious if there could be a new page or section in the docs to list those as a reference?

**Comment by jxom** [comment]
Sure! What is it? I can add a page for community plugins.

**Reply by stevedylandev** [reply]
Awesome! It's a middleware extension of [pinata-fdk](https://github.com/PinataCloud/pinata-fdk) which we would like to call "Pinata Analytics". Have it referenced in our docs on usage but we can make a PR with the blurb on how to use it depending what you want and don't want in the frog docs. https://docs.pinata.cloud/farcaster/fdk#frog-analytics-plug-in

**Reply by tmm** [reply, marked as answer]
Added! https://frog.fm/concepts/middleware#community-middleware
